cluster/down.sh

```diff
@@ -1,22 +1,5 @@
 #!/bin/bash
-#
```

Comment:
either keep license headers in all cluster scripts or drop it from all (i'd rather keep it)